### ISSUE TITLE
Can't disable annoying overlay

### ISSUE THREAD

**Issue by IIARROWS** [tamius-han/ultrawidify#246]
**Describe the bug**
There is an annoying overlay on the top left of the video that says "Aspect ratio controls" every time the mouse enters the video player, and disappears after a second.

**To Reproduce**
Move the mouse inside the video area.

**Expected behavior**
No overlay disrupting the video, or at least show a clear option to disable the behaviour.

**Screenshots**
![image](https://github.com/tamius-han/ultrawidify/assets/14835013/9ca07529-433f-4c18-855d-e12bfd693aef)

**Browser and OS**
 * Browser/browser version: Vivaldi
 * Operating system: Windows 10
 * Ultrawidify version: 6.0.0

**Additional info**
Any other things you want to say.
I have only bad words for whoever came up with this stupid idea, a lot of them. So I won't.

**Comment by clopezit** [comment]
Same issue and complaint as [IIARROWS](https://github.com/IIARROWS) , The new in-player UI behavior is extremely irritating and disruptive. Please allow an option to disable.

**Comment by Hewitt212** [comment]
I agree, the pop is is very annoying, is there really no way to disable it?

**Comment by ilidrissireda** [comment]
Agree with the sentiment, the pop-un is annoying. You can keep your vitriol and bad word allusions to yourself though.

**Comment by IIARROWS** [comment]
I did.
"Any other things you want to say." is part of the template.

**Comment by bnku** [comment]
> Agreed with others, this overlay makes me want to just abandon the extension. I've already disabled it for now since the benefits of the extension are outweighed by the annoyance of the overlay.

in chromium you can disable it by default until you click on its icon. waiting for the ux fix...

**Comment by trelensis** [comment]
Same here, is set the extension on "click to enable" while waiting for a fix.

**Comment by tamius-han** [comment]
> I agree, the pop is is very annoying, is there really no way to disable it?

Not _yet_. The story goes like so:
* google started deprecating manifest v2 extensions a week ago
* my "free time and other obligations" timeline did not agree with that schedule for the first half of this year
* the idea was to get the manifest v3 out _before_ Google gets annoyed, especially (historically) Google's extension review usually took about a business week, and do less critical work _later_ down the line. Theory being: minor annoyance for a week is better than no functionality at all, especially since UI is set to auto-hide as soon as mouse stops moving.

The change should have been rolled back as of about two hours ago, too, though auto-updating extensions doesn't happen that often.

**Comment by clopezit** [comment]
Thanks for the quick rollback. 
The issue with the auto-hide is in theory it works if you open a video and just watch said video. I and I assume others might open something like Youtube or scroll a Twitter feed and still be moving our cursor about as the video plays because we multitask. This is turn as you know leaves a big UI constantly popping up and disappearing over the video continuously.

**Comment by IIARROWS** [comment]
>   ... especially since having in-page UI was something that has been requested by people on more than one occasion so far.

If you want in-page UI, I suggest you to look after Sponsor Block, which puts its icon and menu on the control bar.

**Comment by Hewitt212** [comment]
> > I agree, the pop is is very annoying, is there really no way to disable it?
> 
> Not _yet_. The story goes like so:
> 
> * google started deprecating manifest v2 extensions a week ago
> * my "free time and other obligations" timeline did not agree with that schedule for the first half of this year
> * the idea was to get the manifest v3 out _before_ Google gets annoyed, especially (historically) Google's extension review usually took about a business week, and do less critical work _later_ down the line. Theory being: minor annoyance for a week is better than no functionality at all, especially since UI is set to auto-hide as soon as mouse stops moving.
>   ... especially since having in-page UI was something that has been requested by people on more than one occasion so far.
> 
> The change should have been rolled back as of about two hours ago, too, though auto-updating extensions doesn't happen that often.

Thanks, man. I didn't mean to sound snarky about it, it was just very jarring. Appreciate the work you do!

**Comment by tamius-han** [comment]
> Thanks for the quick rollback. The issue with the auto-hide is in theory it works if you open a video and just watch said video. I and I assume others might open something like Youtube or scroll a Twitter feed and still be moving our cursor about as the video plays because we multitask. This is turn as you know leaves a big UI constantly popping up and disappearing over the video continuously.

In theory, the popup should only show when mouse cursor hovered over the video, and not when hovering elsewhere on the page. In practice, that appears to have stopped working in the last few days... 

> If you want in-page UI, I suggest you to look after Sponsor Block, which puts its icon and menu on the control bar.

Sponsorblock can do that because it has the luxury of only having to need on Youtube. I don't, which means I also can't quite afford to do this.

**Comment by KoenLemmen** [comment]
Thanks for the explanation and fast rollback, also the new GUI that we had briefly before the rollback looks very good, though I do agree with the sentiment here. Looking forward to where this is going. Great extension for the UW gang.
